### PR TITLE
Replace Some Lower in AutoTune Unit Tests

### PR DESCRIPTION
**Pull request by zhhsplendid**
### PR types

Others
### PR changes

Others
### Description

pcard-75206
Replace Some Lower in AutoTune Unit Tests

### CODE REVIEW

File: paddle/cinn/ir/test/collect_ir_nodes_test.cc [hunk 1]
```diff
@@ -42,15 +42,15 @@ TEST(CollectIRNodes, basic) {
...
+  ast_gen_ius::TensorGroup tensor_group({C});
 
-  auto fn = Lower("fn", stages, {A, B, C});
+  auto fn = LowerToAst("fn", {A, B, C}, &tensor_group);
 
   LOG(INFO) << "fn:\n" << fn;
 
   auto tensors =
       CollectIRNodes(fn, [](const Expr* x) { return x->as_tensor(); });
-  ASSERT_EQ(tensors.size(), 5UL);
```
Comment (BiynXu) [inline paddle/cinn/ir/test/collect_ir_nodes_test.cc:53]: Why does the number of tensors decrease here?
It looks like there should be 3 here, but why did the old version have 5 tensors?

Comment (zhhsplendid) [reply paddle/cinn/ir/test/collect_ir_nodes_test.cc:53]: 之前Lower/Ast生成的tensor，会含有计算body，但这个其实我们Lower之后用不着了。

因为计算的tensor在我现在Lower/Ast没了，所以那里改成3了

**Review by BiynXu — APPROVED** [review]
LGTM

**Review by ZzSean — APPROVED** [review]
LGTM

**Pull request merged**